packages/mobile/src/fiatExchanges/ReviewFees.tsx

```diff
@@ -0,0 +1,210 @@
+import Touchable from '@celo/react-components/components/Touchable'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import CurrencyDisplay from 'src/components/CurrencyDisplay'
+import Dialog from 'src/components/Dialog'
+import { CurrencyCode } from 'src/config'
+import { Namespaces } from 'src/i18n'
+import InfoIcon from 'src/icons/InfoIcon'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { navigateToURI } from 'src/utils/linking'
+
+interface Props {
+  provider: string
+  currencyToBuy: CurrencyCode
+  localCurrency: LocalCurrencyCode
+  crypto: {
+    amount: number
+  }
+  fiat: {
+    subTotal: number
+    total: number
+  }
+  feeWaived: boolean
+  feeUrl: string
+}
+
+export default function ReviewFees({
```

Comment:
Just  thought, might not be necessary, but would we like analytics for the dialogs opening in this screen and the `openProviderFeeUrl` function?

Reply:
I'm going to add a separate issue to the backlog to instrument analytics more thoroughly. I think there are a couple of places where this is lacking in the flow so I'll address it all at once :)